sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusSenderClient.java

```diff
@@ -152,4 +208,33 @@ public void send(ServiceBusMessageBatch batch) {
     public void close() {
         asyncClient.close();
     }
+
+    /**
```

Comment:
again moved, I did put friends together, like all send together and other except this one.  
Although all the transaction related messages are friend but they can not be ordered since their name are create, commit, rollback. So I decided to take an exception and kept all transaction related API together.